cli/proc_state.rs

```diff
@@ -90,7 +91,7 @@ pub struct Inner {
   pub shared_array_buffer_store: SharedArrayBufferStore,
   pub compiled_wasm_module_store: CompiledWasmModuleStore,
   pub parsed_source_cache: ParsedSourceCache,
-  pub maybe_resolver: Option<Arc<CliResolver>>,
+  pub resolver: Arc<CliResolver>,
```

Comment:
Awesome, very welcome change, it was super unclear when this resolver is being used. One nitpick though (that we can do another time); maybe we could name it `CliGraphResolver`? I think it might lead to some confusion why there's `ProcState::resolve()` and `ProcState::resolver`

Reply:
Good idea. I often have to remind myself that it's only for the module graph